### PR TITLE
[BMC Helix] Integration support group issue

### PR DESCRIPTION
**Pull request by enes-oezdemir**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Integration has assigned_group as a defined argument but it is in fact expecting assigned_support_group_name in the integration code. 

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by jlevypaloalto — COMMENTED** [review]
Hi @contributor_user,
Thank you for your contribution!

Good work, just one comment: When changing mismatched arguments, lets change the argument name in the code instead of the yml file.

Please feel free to reach out to me with any questions - I'm available here or on slack :)
Thanks again

**Comment by enes-oezdemir** [comment]
> Hi @enes-oezdemir, Thank you for your contribution!
> 
> Good work, just one comment: When changing mismatched arguments, lets change the argument name in the code instead of the yml file.
> 
> Please feel free to reach out to me with any questions - I'm available here or on slack :) Thanks again

done

**Review by jlevypaloalto — APPROVED** [review]
Looks great!

Please merge from master as `circleci` has changed.

**Comment by enes-oezdemir** [comment]
> Looks great!
> 
> Please merge from master as `circleci` has changed.

done

**Comment by jlevypaloalto** [comment]
Hi @enes-oezdemir, the code looks good!

We're ready for a demo. Please check [this page](https://xsoar.pan.dev/docs/contributing/demo-prep), and let me know when you're available for one over DFIR. 
Feel free also to send me a recording of a demo.

File: Packs/BmcITSM/ReleaseNotes/1_0_23.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### BMC Helix ITSM
+
+- Fixed an issue where the integration failed to create incident with assigned support groups.
```
Comment (ShirleyDenkberg) [inline Packs/BmcITSM/ReleaseNotes/1_0_23.md:6]:
```suggestion
Fixed an issue where the integration failed to create incidents with assigned support groups.
```

**Comment by ShirleyDenkberg** [comment]
@jlevypaloalto Doc review completed.

**Comment by jlevypaloalto** [comment]
Closing this PR as it's changes are incorporated in [this](https://github.com/demisto/content/pull/34741) PR.

**Pull request closed without merging**